lib/buffer.js

```diff
@@ -157,6 +157,12 @@ function palloc(that, length) {
   var buf = sliceOnto(allocPool, that, start, end);
   poolOffset = end;
 
+  // Ensure aligned slices
+  if (poolOffset & 0x7) {
+    poolOffset |= 0x7;
```

Comment:
A less convoluted way of writing that is `poolOffset = (poolOffset & ~7) + 8` :-)